src/main/resources/application.properties

```diff
@@ -21,22 +21,23 @@ eurekaclinical.userwebapp.callbackserver = https://localhost:8443
 eurekaclinical.userwebapp.url=https://localhost:8443/eurekaclinical-user-webapp
 eurekaclinical.userservice.url=https://localhost:8443/eurekaclinical-user-service
 
-cas.url=https://localhost:8443/cas-server
+cas.url=https://localhost:4200/cas-server
```

Comment:
Keep port 8443 here, and override it with port 4200 in the embedded tomcat application.properties.